W-12992484 traffic-management-local-mode-gr (#329)
mulesoft/docs-gateway: gateway/1.5/modules/ROOT/pages/flex-gateway-publish-apis-conn.adoc

endif::[]
:imagesdir: ../assets/images

> For Flex Gateway running in Connected Mode, you publish both API instances with single upstream services and with multiple upstream services by using *API Manager*. 

Reviewer: ```suggestion
For Flex Gateway running in Connected Mode, publish both API instances that contain single upstream services and API instances that contain multiple upstream services by using *API Manager*. 
```

Reviewer: I am not sure if "that contain" is correct but "with" is not clear.

> For how to publish Flex Gateway API instances with API Manager, see xref:api-manager::create-instance-task-flex.adoc[].

Reviewer: ```suggestion
For more information, see xref:api-manager::create-instance-task-flex.adoc[].
```

Reviewer: Because the "more information" text comes right after you talk about publishing, you don't need to elaborate on what the information is for.
